src/pages/components/progress.mdx

```diff
@@ -75,6 +75,10 @@ import SizeCode from '!!raw-loader!../../examples/loaders/progress/ProgressSize.
 
 <Component components={props.data.mdx.components} componentName="progress" />
 
+A Progress provides a measurable indicator for uploading and downloading content.
+Nest a Progress within a [FileList](/components/file-upload#files) to indicate
```

Comment:
Technically, the `Progress` can be added to a `File`, not a `FileList`.

Reply:
Good catch, will update!